src/cloudant/document.py

```diff
@@ -341,12 +341,13 @@ def __enter__(self):
 
         return self
 
-    def __exit__(self, *args):
+    def __exit__(self, exc_type, exc_value, traceback):
         """
-        Support context like editing of document fields.  Handles context exit
-        logic.  Executes a Document.save() upon exit.
+        Support context like editing of document fields. Handles context exit
+        logic. Executes a `Document.save()` upon exit iff no exception occurred.
```

Comment:
iff -> if

Reply:
I was abbreviating _if and only if_ but I guess plain old "if" works and will cause less confusion.